Resources/Prototypes/Recipes/Lathes/electronics.yml

```
@@ -604,3 +604,18 @@
+  materials:
+    Steel: 100
+    Glass: 500
+
+- type: latheRecipe
+  id: MachineBluespaceHarvesterCircuitboard
+  result: MachineBluespaceHarvesterCircuitboard
+  completetime: 10
+  materials:
+    Steel: 100
+    Glass: 900
+    Gold: 300     
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)